examples/JoystickControl/JoystickControl.ino

```diff
@@ -0,0 +1,56 @@
+#include 
+Servo servo1;     // 2 Servos Initialized 
+Servo servo2;
+int x_key = A1;     //Horizontal pin for Joystick                                         
+int y_key = A0;     //Vertical pin for Joystick                                         
+int x_pos;
+int y_pos;
+int servo1_pin = 8;   //Signal Pins for Servo
+int servo2_pin = 9;  
+int initial_position = 90;    //Initially Servos are moved to 90
+int initial_position1 = 90;
+
+void setup ( ) {
```

Comment:
```suggestion
void setup() {
```
Please study the official example sketches and follow the style conventions we have established.